src/controller/CHIPDeviceController.cpp

```diff
@@ -1614,6 +1614,20 @@ void OnBasicFailure(void * context, CHIP_ERROR error)
     commissioner->CommissioningStageComplete(error);
 }
 
+void NonConcurrentTimeout(void * context, CHIP_ERROR error)
```

Comment:
Should be static or in an anonymous namespace, right?  I know OnBasicSuccess and OnBasicFailure are neither; we should fix that.

Reply:
Updated